contracts/StablecoinConverter.sol

```diff
@@ -41,6 +41,7 @@ contract StablecoinConverter is EpochTokenLocker {
     mapping(address => Order[]) public orders;
 
     IdToAddressBiMap.Data private registeredTokens;
+
     uint public MAX_TOKENS; // solhint-disable-line
```

Comment:
Would prefer to know what this is disabling...

I believe you can write the specific item right after.

Reply:
Yes, it is needed for testing. If we want to shoot 2**16 addToken requests for testing the limit, it takes too much time.